caper/cromwell_workflow_monitor.py

```diff
@@ -237,7 +247,6 @@ def _update_subworkflows(self, stderr):
     def _update_tasks(self, stderr):
         """Check if workflow's task status changed by parsing Cromwell's stderr lines.
         """
-        updated_workflows = set()
         for line in stderr.split('\n'):
             r_common = None
             r_start = re.findall(CromwellWorkflowMonitor.RE_TASK_START, line)
```

Comment:
Same comment about `if r:` etc. as before.

Reply:
Fixed as you suggested.